### PR TITLE
fix: refactor input box (DE44413)

### PR DESCRIPTION
**Pull request by devpow112**
This replaces the swapping of a text box and label to handle input with a single input box styled to look like a label when not in focus. This will allow it to function on Firefox and also not have issues with VoiceOver (thanks again @semibran for testing that part). Will also take care of **DE44403**.

Best viewed with [white-space diff off](https://github.com/Brightspace/d2l-rubric/pull/908/files?w=1).

### CODE REVIEW

**Comment by devpow112** [comment]
I'm still doing some testing so sorry if more changes drop. Mainly testing accessibility. I don't expect any large changes.

**Comment by devpow112** [comment]
> I'm still doing some testing so sorry if more changes drop. Mainly testing accessibility. I don't expect any large changes but I'm expecting some changes.

I think that's it. Using NVDA it seems to be the same behaviour before and after my change.,

File: d2l-rubric-editable-score.js [hunk 1]
```diff
@@ -304,12 +345,16 @@ Polymer({
...
 		this.overriddenStyling = this._shouldShowOverrideState(this.scoreOverridden, this.readOnly, this.hovered);
 
 		if (this.scoreOverridden && this.hovered === -1 && !this.totalScore) {
 			this.score = this.overriddenScore;
 		}
+
+		this.scoreWidth = this.score ? this.score.toString().length : 1;
+		this.scoreFractional = this.score ?
+			Number(this.score) === this.score && this.score % 1 !== 0 :
+			false;
```
Comment (semibran) [inline d2l-rubric-editable-score.js:357]: super small nit, but.....
```suggestion
		this.scoreFractional = typeof this.score === "number" && this.score % 1 !== 0;
```

**Review by semibran — APPROVED** [review]
Looks fine but here's hoping the important stuff is still 1:1 in test 🤞

**Review by semibran — APPROVED** [review]

**Pull request merged**